src/server/game/Entities/Player/Player.cpp

```diff
@@ -7534,6 +7534,16 @@ void Player::SendLoot(ObjectGuid guid, LootType loot_type)
     PermissionTypes permission = ALL_PERMISSION;
 
     LOG_DEBUG("loot", "Player::SendLoot");
+
+    const AuraType toRemove[] = {SPELL_AURA_MOD_STEALTH, SPELL_AURA_MOD_INVISIBILITY, SPELL_AURA_FEIGN_DEATH};
```

Comment:
maybe use `std::vector` ?

Reply:
Done, though I'm not sure I initialized that properly, I never use vectors for stuff that's known ahead of time.

Reply:
Hmm, need learn this, maybe need use `constexpr std::array`

Reply:
Yes, need to use `std::array`, thanks for the new knowledge